### ISSUE TITLE
TypeError: object is not a function when running example from readme

### ISSUE THREAD

**Issue by jaybaker** [romeovs/gulp-browserify#2]
I get an error trying the example from the readme. I'm not sure the exports are defined correctly.

This seems like a great "wrapper" for browserify. Is it working?

**Comment by romeovs** [comment]
Yes this was a documentation issue! Thanks for pointing it out, I've corrected it. You should use `import Browserify from 'browserify-gulp'` or `var Browserify = require('browserify-gulp').default` if you're not using the ES6 import syntax.

**Comment by romeovs** [comment]
Did this fix your issue?
